### PR TITLE
[openwebnet] Fix Things synchronization at boot / reconnect

### LINKED ISSUE #11965: [openwebnet] USB rollershutters stop working after changing .items file
When a REFRESH is sent to a USB (BTicino zigbee) shutter, for example after modifying an .items file, requests sent to the USB gateway are not processed anymore.
The problem is present only for older gateways (firmware <= 1.2.3) , which are affected by a bug when answering to state request (no ACK is returned).

This can be solved by solving this issue in the openwebnet4j lib: see mvalla/openwebnet4j#27

### LINKED ISSUE #11972: [openwebnet] shutters are not refreshed at boot
Shutter items/things are not refreshed at boot and the following exception is returned 

```
00:18:21.059 DEBUG [OH-thingHandler-14  ] [andler.OpenWebNetBridgeHandler:396  ] - Refreshing all devices for bridge openwebnet:bus_gateway:mybridge
00:18:21.061 WARN  [OH-thingHandler-14  ] [rappedScheduledExecutorService:67   ] - Scheduled runnable ended with an exception: java.lang.IllegalArgumentException: UID must have at least 4 segments.
	at org.openhab.core.common.AbstractUID.<init>(AbstractUID.java:71)
	at org.openhab.core.common.AbstractUID.<init>(AbstractUID.java:49)
	at org.openhab.core.thing.UID.<init>(UID.java:48)
	at org.openhab.core.thing.ChannelUID.<init>(ChannelUID.java:51)
	at org.openhab.binding.openwebnet.internal.handler.OpenWebNetAutomationHandler.refreshDevice(OpenWebNetAutomationHandler.java:174)
	at org.openhab.binding.openwebnet.internal.handler.OpenWebNetBridgeHandler.refreshAllDevices(OpenWebNetBridgeHandler.java:400)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
```

### LINKED ISSUE #11974: [openwebnet] some items/things do not come online after boot
In systems with many Things configured (>20), at boot or when the BUS gateway is re-enabled, some things do not get online and their state is not updated.

## Expected Behavior
All things configured come online and their state is updated

## Current Behavior
Some things are not set online, their state is not updated.

## Possible Solution
`OpenWebNetBridgeHandler.refreshAllDevices()`  is called too early: some Things are not yet associated to the bridge or are not initialized yet, so their state is not updated. Solution is to delay the call to refreshAllDevices.

## Steps to Reproduce (for Bugs)
Configure >20 things, disable the bridge and then re-enable it: some things do not come online

### PR DESCRIPTION
**Pull request by mvalla**
This PR solves some issues related to Things synchronization when the associated bridge is initialized, for example at boot,  when it is re-enabled or when it reconnects to a BTicino gateway. In systems with many Things configured (+100), not all were synchronized properly and put ONLINE/OFFLINE.
Fixes: #11627. Fixes #11972. Fixes #11974.
It also fixes #11965: USB rollershutters blocked when a REFRESH command was sent to them.

openwebnet4j lib updated to latest: 0.7.1

Several tests performed successfully by power users on real systems with many Things configured (+100 things).

### CODE REVIEW

**Comment by mvalla** [comment]
@lolodomo . This PR is no more WIP and is ready for review as it has been tested by users. 
Since it fixes 4 bugs, I think is better to change its label to bug instead of enhancement (I also changed its title accordingly).

File: bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/i18n/openwebnet_it.properties [hunk 1]
```diff
@@ -14,5 +14,6 @@ offline.conf-error-auth = Autenticazione fallita. Verificare la password nella c
 offline.comm-error-disconnected = Disconnesso dal gateway.
 offline.comm-error-timeout = Connessione al gateway non riuscita in tempo.
 offline.comm-error-connection = Impossibile connettersi al gateway.
+offline.comm-error-state = Impossibile aggiornare lo stato.
```
Comment (lolodomo) [inline bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/i18n/openwebnet_it.properties:17]: Please do not update the Italian translation in this PR; do it later in Crowdin.

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetBridgeHandler.java [hunk 2]
```diff
@@ -392,13 +404,62 @@ protected void unregisterDevice(String ownId) {
...
+                // set a check that all things are Online
+                refreshAllSchedule = scheduler.schedule(() -> checkAllRefreshed(things), REFRESH_ALL_CHECK_DELAY_SEC,
+                        TimeUnit.SECONDS);
             }
+        } else {
+            logger.debug("--- --- NO CHILD DEVICE to REFRESH for bridge {}", thing.getUID());
+        }
+    }
+
+    private void checkAllRefreshed(List<@NonNull Thing> things) {
```
Comment (lolodomo) [inline bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetBridgeHandler.java:445]: `@NonNull` not required here too.

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetBridgeHandler.java [hunk 3]
```diff
@@ -392,13 +404,62 @@ protected void unregisterDevice(String ownId) {
...
-    private void refreshAllDevices() {
-        logger.debug("Refreshing all devices for bridge {}", thing.getUID());
-        for (Thing ownThing : getThing().getThings()) {
-            OpenWebNetThingHandler hndlr = (OpenWebNetThingHandler) ownThing.getHandler();
-            if (hndlr != null) {
-                hndlr.refreshDevice(true);
+    private void refreshAllBridgeDevices() {
+        logger.debug("--- --- ABOUT TO REFRESH ALL devices for bridge {}", thing.getUID());
+        int howMany = 0;
+        final List<@NonNull Thing> things = getThing().getThings();
```
[the remark this answers is not in the document]
Comment (mvalla) [reply bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetBridgeHandler.java]: removed

File: bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetThingHandler.java [hunk 4]
```diff
@@ -192,19 +203,87 @@ protected void handleMessage(BaseOpenMessage msg) {
...
+                    refreshDevice(true);
+                } else {
+                    logger.debug("--- refreshAllDevices() : refresh all devices just sent... ({})", thing.getUID());
+                }
+                // sometimes GENERAL (e.g. #*1*0##) refresh requests do not return state for all devices, so let's
+                // schedule another single refresh device, just in case
+                refreshTimeout = scheduler.schedule(() -> {
+                    if (thing.getStatus().equals(ThingStatus.UNKNOWN)) {
+                        logger.debug(
+                                "--- refreshAllDevices() : schedule expired: --UNKNOWN-- status for {}. Rerfreshing it...",
```
[the remark this answers is not in the document]
Comment (mvalla) [reply bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetThingHandler.java]: fixed

File: bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/i18n/openwebnet_it.properties (hunk 1 above)
Comment (mvalla) [reply bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/i18n/openwebnet_it.properties]: removed change

File: bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetBridgeHandler.java (hunk 2 above)
Comment (mvalla) [reply bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetBridgeHandler.java]: removed

**Comment by mvalla** [comment]
thanks @lolodomo for the review, all requested changes have been implemented

**Review by lolodomo — APPROVED** [review]
LGTM

**Pull request merged**